packages/jest-config/src/__tests__/readConfig.test.ts

```diff
@@ -3,8 +3,8 @@
 import {readConfig} from '../index';
 
 test('readConfig() throws when an object is passed without a file path', () => {
-  expect(() => {
-    readConfig(
+  expect(async () => {
```

Comment:
Need to await the `expect` and change matcher to be `rejects.toThrowError`

Reply:
goes for a lot od the test changes

Reply:
Fixed, thanks!